checkers 0.5.5

```
--  While building package checkers-0.5.5 using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.2.0.0_ghc-8.10.1 --builddir=.stack-work/dist/x86_64-linux/Cabal-3.2.0.0 build lib:checkers -
-ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/checkers
-0.5.5.log

    Configuring checkers-0.5.5...
    Preprocessing library for checkers-0.5.5..
    Building library for checkers-0.5.5..
    [10 of 15] Compiling Test.QuickCheck.Checkers

    /var/stackage/work/unpack-dir/unpacked/checkers-0.5.5-6c76c790e87c691cacd58fb03968
dc19585f68eabdf6c17435fc9a5641aacbf9/src/Test/QuickCheck/Checkers.hs:256:30: error:
        Ambiguous occurrence ‘First’
        It could refer to
           either ‘Data.Monoid.First’,
                  imported from ‘Data.Monoid’ at src/Test/QuickCheck/Checkers.hs:(53,1
)-(57,27)
               or ‘Data.Semigroup.First’,
                  imported from ‘Data.Semigroup’ at src/Test/QuickCheck/Checkers.hs:69
:1-21
        |
    256 | instance EqProp a => EqProp (First a)
        |                              ^^^^^

    /var/stackage/work/unpack-dir/unpacked/checkers-0.5.5-6c76c790e87c691cacd58fb03968
dc19585f68eabdf6c17435fc9a5641aacbf9/src/Test/QuickCheck/Checkers.hs:257:30: error:
        Ambiguous occurrence ‘Last’
        It could refer to
           either ‘Data.Monoid.Last’,
                  imported from ‘Data.Monoid’ at src/Test/QuickCheck/Checkers.hs:(53,1
)-(57,27)
               or ‘Data.Semigroup.Last’,
                  imported from ‘Data.Semigroup’ at src/Test/QuickCheck/Checkers.hs:69
:1-21
        |
    257 | instance EqProp a => EqProp (Last a)
        |                              ^^^^
```

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -474,6 +474,7 @@ packages:
         - bsb-http-chunked
         - coercible-utils < 0 # ghc 8.10
         - hspec-parsec
+        - checkers
 
     "Joe M <joe9mail@gmail.com> @joe9":
         - logger-thread < 0 # via protolude
@@ -965,7 +966,7 @@ packages:
     "Mario Blazevic <blamario@yahoo.com> @blamario":
         - monad-parallel
         - monad-coroutine
-        - incremental-parser < 0 # via checkers
+        - incremental-parser
         - monoid-subclasses
         - picoparsec < 0 # BuildFailureException Process exited with ExitFailure 1: ./Setup build
         - rank2classes
@@ -1484,7 +1485,7 @@ packages:
         - ghcjs-perch < 0 # build failure with GHC 8.4
 
     "Tom Ellis <tom-stackage@jaguarpaw.co.uk> @tomjaguarpaw":
-        - opaleye < 0 # via checkers
+        - opaleye
         - product-profunctors
 
     "Samplecount stefan@samplecount.com @kaoskorobase":
@@ -1671,7 +1672,7 @@ packages:
         - getopt-generics
         - graph-wrapper
         - string-conversions
-        - hspec-checkers < 0 # via checkers
+        - hspec-checkers
         - FindBin
 
     "Jan Stolarek <jan.stolarek@p.lodz.pl> @jstolarek":
@@ -4345,7 +4346,6 @@ packages:
         - cereal
         - cereal-text
         - cereal-vector
-        - checkers < 0 # https://github.com/commercialhaskell/stackage/issues/5433
         - chunked-data
         - cipher-aes128 < 0 # via Cabal-3.0.0.0
         - cipher-blowfish < 0 # MonadFail
@@ -4526,7 +4526,7 @@ packages:
         - monads-tf
         - mono-traversable-instances
         - mono-traversable-keys
-        - multiset < 0 # via checkers
+        - multiset
         - mwc-random
         - names-th
         - nanospec

